contracts/src/everlong/EverlongAdmin.sol

```diff
@@ -1,12 +1,16 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity 0.8.20;
 
-import { IEverlongAdmin } from "../interfaces/IEverlongAdmin.sol";
+import { IEverlongAdmin } from "contracts/src/interfaces/IEverlongAdmin.sol";
```

Comment:
Why are we making these absolute? Everywhere else in the project, we use relative for contracts in `contracts/src` and absolute elsewhere. The reason why we do this is annoying and historical, but I don't want to change unless we change all of the files.

Reply:
Got it, didn't make the connection on where rel/abs paths were being used, will put it back to relative